### PR TITLE
Check that charge prices were added to db in pipeline test

### PR DESCRIPTION
**Pull request by HenrikSommer**
With this PR we ensure that charge prices are added to the database when a charge is saved.

## References
#323

### CODE REVIEW

**Review by BjarkeMeier — COMMENTED** [review]
Why is this part of the integration test rather than a -- much simpler to maintain -- repository test?

**Review by Mech0z — APPROVED** [review]

**Comment by HenrikSommer** [comment]
> Why is this part of the integration test rather than a -- much simpler to maintain -- repository test?

Good question! - we defined story #323 three months ago, but maybe it is just not relevant anymore?

**Review by Mech0z — APPROVED** [review]

**Comment by BjarkeMeier** [comment]
> > Why is this part of the integration test rather than a -- much simpler to maintain -- repository test?
> 
> Good question! - we defined story #323 three months ago, but maybe it is just not relevant anymore?

I would prefer a database unit test (and an update of the wording in the story)...

**Review by lasrinnil — APPROVED** [review]

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/Application/ChangeOfCharges/ChangeOfChargePipelineTests.cs [hunk 1]
```diff
@@ -95,18 +96,19 @@ public ChangeOfChargePipelineTests([NotNull] DbContextRegistrator dbContextRegis
...
             // assert
-            Assert.True(changeOfChargesMessageResult.IsSucceeded);
-            Assert.Equal(chargeCommand.ChargeOperation.Id, receivedEvent.OriginalTransactionReference);
-            Assert.NotNull(receivedEvent);
-            Assert.True(chargeExists);
+            changeOfChargesMessageResult.IsSucceeded.Should().BeTrue();
+            chargeCommand.ChargeOperation.Id.Should().Be(receivedEvent.OriginalTransactionReference);
+            receivedEvent.Should().NotBeNull();
+            charge.Should().NotBeNull();
+            charge.Points.Should().NotBeNullOrEmpty();
```
Comment (prtandrup) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/Application/ChangeOfCharges/ChangeOfChargePipelineTests.cs:111]: How about making this an explicit assert, checking that the price was in fact saved as provided in the test file?

Btw, I do see some values in the test file used, which seems postman related, e.g. 
"Time": "{{$isoTimestampPlusOneMonth}}" - Should we edit those values? If you'd like, I might have time for that today.

**Comment by prtandrup** [comment]
> > > Why is this part of the integration test rather than a -- much simpler to maintain -- repository test?
> > 
> > 
> > Good question! - we defined story #323 three months ago, but maybe it is just not relevant anymore?
> 
> I would prefer a database unit test (and an update of the wording in the story)...

👍 for repo unit test
I'll like to keep this pipeline test in place. And instead raise the question whether this is an integration test or actually the beginning of how we can test flows throughout our domain. I will put a ticket up for our next dev session.

**Review by prtandrup — APPROVED** [review]

**Pull request merged**